Aurora/Examples/ConsoleExample/StdioLogSystem.cpp

```diff
@@ -19,8 +19,8 @@
 
 // Currently we create the file on first real output, and only close it on shutdown
 
-#include <stdarg.h>
-#include <assert.h>
+#include <cstdarg>
+#include <cassert>
```

Comment:
Please remove these changes, they are not required and the C++ versions are known to increase compile times.